fr/views/helpers/form.rst

```diff
@@ -44,33 +44,30 @@ ce que nous allons voir dans la prochaine section. Ceux intégrés correspondent
 aux valeurs suivantes de ``$context``:
 
 * Une instance ``Entity`` ou un iterateur qui mappe vers
-  `EntityContext <https://api.cakephp.org/3.x/class-Cake.View.Form.EntityContext.html>`_;
-  ce contexte permet au FormHelper de fonctionner avec les retours de l'ORM.
+  `EntityContext <https://github.com/cakephp/cakephp/tree/master/src/View/Form/EntityContext.php>`_;
+  ce contexte permet au FormHelper de fonctionner avec les retours de l'ORM
+  intégré.
 
 * Un tableau contenant la clé ``schema``, qui mappe vers
-  `ArrayContext <https://api.cakephp.org/3.x/class-Cake.View.Form.ArrayContext.html>`_
+  `ArrayContext <https://github.com/cakephp/cakephp/tree/master/src/View/Form/ArrayContext.php>`_
```

Comment:
These should use `4.x` for the api docs url:

https://api.cakephp.org/4.x/class-Cake.View.Form.ArrayContext.html

Reply:
Actually, its english source should also.